AppHandling/Compile-AppInNavContainer.ps1

```diff
@@ -485,6 +485,14 @@ try {
                     }
                     if ($throw) {
                         Write-Host "ERROR $($_.Exception.Message)"
+                        if ($_.Exception.InnerException)
+                        {

```

Comment:
Isn't these things part of GetExtendedErrorMessage as well?

Reply:
Maybe truncated ? But without this part, I didn't had the reason of failure (only the fact that there was an issue with SSL channel and the trace)

I'll double check - but with this change, it disclose up to the fact issue is related to not trusted certificate.

Reply:
Just had a quick look in it
It might be due to the level of the exception from which you get enough details when an SSL exception occurred (third one here)

As per what I understand from the current GetExtendedErrorMessage - it goes only up to the second level.

By looking on other points, I'll see if I cannot simply enhance this method instead.